src/frontends/onnx/frontend/src/op/reduce.cpp

```diff
@@ -158,6 +162,30 @@ ov::OutputVector reduce_sum_square(const ov::frontend::onnx::Node& node) {
 }
 
 }  // namespace set_1
+
+namespace set_11 {
+ov::OutputVector reduce_l2(const Node& node) {
+    if (node.get_ov_inputs().at(0).get_partial_shape().rank().is_static()) {
+        auto input_rank = node.get_ov_inputs().at(0).get_partial_shape().rank().get_length();
+        return {
+            make_ng_reduction_op<v4::ReduceL2>(node,
+            node.get_ov_inputs().at(0), true, {-input_rank, input_rank - 1})};
+    }
+    return {make_ng_reduction_op<v4::ReduceL2>(node, node.get_ov_inputs().at(0))};
+    }
+}  // namespace set_11
+
+namespace set_13 {
+ov::OutputVector reduce_sum(const ov::frontend::onnx::Node& node) {
+    return {make_ng_reduction_op<v1::ReduceSum>(node, node.get_ov_inputs().at(0), false)};
+}
+}  // namespace set_13
+
+namespace set_18 {
+ov::OutputVector reduce_l2(const ov::frontend::onnx::Node& node) {
+    return {make_ng_reduction_op<v4::ReduceL2>(node, node.get_ov_inputs().at(0))};
```

Comment:
Overall - good, just add missing checks

Reply:
> missing checks

I'm checking [ONNX documentation](https://onnx.ai/onnx/operators/onnx__ReduceL2.html#reducel2) to understand what additional checks I should perform. I don't understand which ones you're referring to. Could you please be more specific about it?